llvm/lib/Target/AMDGPU/AMDGPUCallLowering.cpp

```diff
@@ -127,10 +130,9 @@ struct AMDGPUIncomingArgHandler : public CallLowering::IncomingValueHandler {
       unsigned CopyToBits = 32;
 
       // When function return type is i1, it may be in a 64b register.
-      if (VA.getLocVT() == MVT::i1) {
-        if (MIRBuilder.getMF().getSubtarget<GCNSubtarget>().isWave64())
-          CopyToBits = 64;
-      }
+      if (VA.getLocVT() == MVT::i1 &&
+          MIRBuilder.getMF().getSubtarget<GCNSubtarget>().isWave64())
+        CopyToBits = 64;
```

Comment:
Ternary operator in initialization of CopyToBits

Reply:
Done.